jsonlogic_test.go

```diff
@@ -184,6 +184,441 @@ func TestSimpleVarComparison(t *testing.T) {
 	}
 }
 
+func TestSafeToBoolValid(t *testing.T) {
+	var a interface{}
+	a = true
+	pa, err := safeToBool(a)
+
+	if err != nil {
+		t.Fatal("Should not have thrown error, it's a valid bool value")
+	}
+	if pa == false {
+		t.Fatal("Should not have the default value")
+	}
+}
+
+func TestSafeToBoolString(t *testing.T) {
+	var a interface{}
+	a = "true"
+	pa, err := safeToBool(a)
+
+	if err != nil {
+		t.Fatal("Should not have thrown error, it's a valid bool value")
+	}
+	if pa == false {
+		t.Fatal("Should not have the default value")
+	}
+}
+
+func TestSafeToBoolEmpty(t *testing.T) {
+	var a interface{}
+	pa, err := safeToBool(a)
+
+	if err == nil {
+		t.Fatal("The result expected must be false, nil value")
```

Comment:
"the expected result is false or a nil value"

Reply:
Corregido.